.atomist/tests/NewRugArchiveProject.rt

```diff
@@ -32,8 +32,7 @@ then
     and fileContains ".atomist/manifest.yml" version
     and fileContains ".atomist/manifest.yml" project_name
     and fileContains ".atomist/manifest.yml" group_id
-    and fileExists ".atomist/editors/AddReadme.rug"
-    and fileExists ".atomist/tests/AddReadme.rt"
+    and { !result.fileContains("README.md", "creation_date") }
```

Comment:
This is likely passing because "creation_date" does not appear in the original README. A template README is no longer used.